lib/tools.js

```diff
@@ -278,17 +279,45 @@ function getMac(callback) {
     });
 }
 
+// Is docker environment
+function isDocker() {
+    let _isDocker = false;
+    try {
+        fs.statSync('/.dockerenv');
+        _isDocker = true;
+    } catch {
+
+    }
+
+    let _isDockerGroup = false;
+
+    try {
+        _isDockerGroup = fs.readFileSync('/proc/self/cgroup', 'utf8').includes('docker');
+    } catch {
+
```

Comment:
```suggestion
   // ignore
```

otherwise eslint is unhappy and ci gets red